declarative/launcher/applicationmanager.cpp

```diff
@@ -59,20 +59,24 @@ QList<Application *> ApplicationManager::applications() const
 {
     QList<Application *> apps;
 
-    Q_FOREACH (Application *app, m_apps) {
+    Q_FOREACH (Application *app, m_apps.values()) {
```

Comment:
there is code using Q_FOREACH all over the repo for sure as it was the reccomended way just added a task to remember that we have to go through the code and change https://github.com/qmlos/shell/issues/52